mlflow/server/handlers.py

```diff
@@ -88,6 +89,7 @@
 _model_registry_store = None
 _artifact_repo = None
 STATIC_PREFIX_ENV_VAR = "_MLFLOW_STATIC_PREFIX"
+_mime_type = mimetypes.MimeTypes()
```

Comment:
Do we need this? Isn't the func `guess_type()` in the `__init__()` of the `mimetypes` package?
Could we just import this function directly `from mimetypes import guess_type`?